src/net/sf/freecol/client/gui/panel/CargoPanel.java

```diff
@@ -49,25 +48,23 @@ public class CargoPanel extends FreeColPanel
 
     private DefaultTransferHandler defaultTransferHandler = null;
 
-
     /**
      * Creates this CargoPanel.
      *
      * @param freeColClient The {@code FreeColClient} for the game.
-     * @param withTitle Should the panel have a title?
+     * @param withTitle     Should the panel have a title?
      */
     public CargoPanel(FreeColClient freeColClient, boolean withTitle) {
         super(freeColClient, "CargoPanelUI",
-              new MigLayout("wrap 6, fill, insets 0"));
+                new MigLayout("wrap 6, fill, insets 0"));
 
         this.carrier = null;
-        this.defaultTransferHandler
-            = new DefaultTransferHandler(getFreeColClient(), this);
+        this.defaultTransferHandler = new DefaultTransferHandler(getFreeColClient(), this);
 
-        if (withTitle) setBorder(Utility.localizedBorder("cargoOnCarrier"));
+        if (withTitle)
+            setBorder(Utility.localizedBorder("cargoOnCarrier"));
```

Comment:
I prefer always adding braces ... and especially when having the statement on a different line ... like this:
```
        if (withTitle) {
            setBorder(Utility.localizedBorder("cargoOnCarrier"));
        }
```

Reply:
Fixed.